sources/packages/backend/libs/services/src/clamav/services/clamav.service.ts

```diff
@@ -22,27 +27,25 @@ export class ClamAVService {
   /**
    * Scans the file for viruses.
    * @param stream stream to be scanned.
-   * @param processSummary process summary logs.
    * @returns boolean true if the stream is virus infected,
    * false if the stream is not virus infected, and
    * null if the stream could not be scanned for viruses.
    */
-  async scanFile(
-    stream: Readable,
-    processSummary: ProcessSummary,
-  ): Promise<boolean> {
+  async scanFile(stream: Readable): Promise<VirusScanCode> {
```

Comment:
`VirusScanCode` is misleading since the core idea of the method result is to determine if the file has a virus or not.
Either way, having the interface return the positive result `isInfected` and a possible error code does not seem needed.
Can we communicate the error using a throw?

Reply:
It is no longer a problem since the code was refactored hence closing it.